woocommerce/woocommerce-ios
# Remove magic link emphasis during the WPCom login flow

Part of #7437


## Description

According to an internal chat [p1677660036646379/1677650404.493279-slack-C046HDZL87J] - the magic link emphasis experiment was removed from Android but still left running on iOS. This PR disables the feature flag to always prioritize password login over magic link login.

## Testing instructions

- Log out of the app or skip onboarding if needed.
- On the prologue screen select Enter your site address.
- Enter the address of a Jetpack site.
- Enter the email address of a WPCom account that has password.
- Notice that the password screen is displayed immediately afterward. The magic link option is available as a secondary button at the bottom of the screen.

## Screenshots

<img src="https://user-images.githubusercontent.com/5533851/223939334-97c79eec-56db-45e8-a8ed-6d7307f028ff.jpg" width=320 />


---
